coalib/bearlib/abstractions/Linter.py

```diff
@@ -701,7 +701,7 @@ def linter(executable: str,
 
         - ``RESULT_SEVERITY.MAJOR``: Mapped by ``error``.
```

Comment:
hey, the commit message does not give enough info. please change the shortlog or add a better explanation to the body to show why this is needed.